cyclonedx_py/parser/conda.py

```diff
@@ -64,7 +64,7 @@ def _conda_packages_to_components(self) -> None:
         for conda_package in self._conda_packages:
             purl = conda_package_to_purl(conda_package)
             c = Component(
-                name=conda_package['name'], version=conda_package['version'],
+                name=conda_package['name'], bom_ref=purl.to_string(), version=str(conda_package['version']),
```

Comment:
:question: 
could you explain, why the string cast was necessary? 
not that it is wrong, but i am worried whether there should be a regression test that failes when the string cast gets removed.

Reply:
I didn't add that - although it looks like I may have taken it out in a previous commit, and added it back in here! I'll take it out again.

Original code is here, which contains the cast, added by @madpah: https://github.com/CycloneDX/cyclonedx-python/commit/7bb6d328adec59cdd4c3ab80eb5f39568ca3bc9c#diff-97c8c273a746672bb6c4a9fa6e5595010c2b9b7efca301ae946946dc412af924